extensions/wikia/WikiFactory/Close/maintenance.php

```diff
@@ -7,7 +7,11 @@
  * @author Krzysztof Krzyżaniak <eloy@wikia-inc.com>
  */
 
-ini_set( "include_path", dirname(__FILE__)."/../../../../maintenance/" );
+use Swagger\Client\Discussion\Api\SitesApi;
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\Swagger\ApiProvider;
```

Comment:
That said, if this works as-is (possibly `require_once`s are handled before `use`s) then it doesn't need to move.  How did this behave when you ran it?